src/rust/ensogl/lib/core/src/display/scene/layer.rs

```diff
@@ -236,9 +236,9 @@ pub struct LayerModel {
     symbols_ordered                 : RefCell<Vec<SymbolId>>,
     depth_order                     : RefCell<DependencyGraph<LayerItem>>,
     depth_order_dirty               : dirty::SharedBool<OnDepthOrderDirty>,
-    parents                         : Rc<RefCell<Vec<Children>>>,
+    parents                         : Rc<RefCell<Vec<Sublayers>>>,
```

Comment:
I miss it on first review: I don't understand why parents are vector of sublayers? Could you add a comment here?